src/cloud/internal/cloudservice.cpp

```diff
@@ -270,6 +270,9 @@ void CloudService::signOut()
     TRACEFUNC;
 
     QUrl signOutUrl = prepareUrlForRequest(configuration()->loginApiUrl());
```

Comment:
Actually, a bit confusing that the method is called `loginApiUrl` while it returns the logout URL...

Reply:
Yeap, it needs to be fixed as well. I decided to not include refactoring to this PR.